trieTools/zeroBalanceSystemAccountChecker/config.toml

```diff
@@ -0,0 +1,8 @@
+[config]
+    [config.elasticIndexerConfig]
+        url = ""
+        username = ""
+        password = ""
+
+    [config.gateway]
+        url = "https://gateway.elrond.com"
```

Comment:
empty line and remove gateway url from here

Reply:
Fixed